cmd/lnmuxd/server.go

```diff
@@ -154,16 +154,16 @@ func (s *server) SubscribeInvoiceAccepted(req *lnmuxrpc.SubscribeInvoiceAccepted
 	}
 }
 
-func (s *server) WaitForInvoiceSettled(ctx context.Context,
-	req *lnmuxrpc.WaitForInvoiceSettledRequest) (
-	*lnmuxrpc.WaitForInvoiceSettledResponse, error) {
+func (s *server) WaitForInvoiceFinalEvent(ctx context.Context,
+	req *lnmuxrpc.WaitForInvoiceFinalEventRequest) (
+	*lnmuxrpc.WaitForInvoiceFinalEventResponse, error) {
 
 	hash, err := lntypes.MakeHash(req.Hash)
 	if err != nil {
 		return nil, err
 	}
 
-	err = s.settledHandler.WaitForInvoiceSettled(ctx, hash)
+	invoiceStatus, err := s.settledHandler.WaitForInvoiceFinalEvent(ctx, hash)
```

Comment:
`WaitForInvoiceFinalStatus`, to remain consistent with the variable name/status enum?